src/app/space/create/apps/apps.module.ts

```diff
@@ -24,7 +26,8 @@ const imports = USE_RUNTIME_CONSOLE ?
     CollapseModule.forRoot(),
     AccordionModule.forRoot(),
     CommonModule,
-    AppsRoutingModule
+    AppsRoutingModule,
+    ChartModule
```

Comment:
Not sure if this holds true on this sub-component but at the parent level the AppsRoutingModule must appear last.